Specs/DateTime/Spanish/DateTimeModel.json

```diff
@@ -1491,5 +1491,17039 @@
+            {
+              "timex": "(T07:30,T09:30,PT2H)",
+              "type": "timerange",
+              "start": "07:30:00",
+              "end": "09:30:00"
+            },
+            {
+              "timex": "(T19:30,T21:30,PT2H)",
+              "type": "timerange",
+              "start": "19:30:00",
+              "end": "21:30:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "¿dónde estabas entre el 30 de julio y 30 de spetiembre?",
+    "Context": {
+      "ReferenceDateTime": "2019-01-31T00:00:00"
+    },
+    "NotSupported": "javascript, java, python",
+    "Results": [
+      {
+        "Text": "entre el 30 de julio y 30 de spetiembre",
+        "Start": 15,
+        "End": 53,
+        "TypeName": "datetimeV2.timerange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(T07:30,T09:30,PT2H)",
+              "type": "timerange",
+              "start": "07:30:00",
+              "end": "09:30:00"
+            },
+            {
+              "timex": "(T19:30,T21:30,PT2H)",
+              "type": "timerange",
+              "start": "19:30:00",
+              "end": "21:30:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "¿dónde estabas entre 30 jul. y 30 sept.?",
+    "Context": {
+      "ReferenceDateTime": "2019-01-31T00:00:00"
+    },
+    "NotSupported": "javascript, java, python",
+    "Results": [
+      {
+        "Text": "entre 30 jul. y 30 sept.",
+        "Start": 15,
+        "End": 38,
+        "TypeName": "datetimeV2.timerange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(T09:30,T19:30,PT10H)",
+              "type": "timerange",
+              "start": "09:30:00",
+              "end": "19:30:00"
+            },
+            {
+              "timex": "(T21:30,T07:30,PT10H)",
+              "type": "timerange",
+              "start": "21:30:00",
+              "end": "07:30:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "¿dónde estabas entre 30-jul y 30-sept?",
+    "Context": {
+      "ReferenceDateTime": "2019-01-31T00:00:00"
+    },
+    "NotSupported": "javascript, java, python",
+    "Results": []
+  },
+  {
+    "Input": "reserve una reunión para el lunes 21, entre las 9:30 y las 3:00 p.m. pst.",
+    "Context": {
+      "ReferenceDateTime": "2019-02-27T00:00:00"
+    },
+    "NotSupported": "javascript, java, python",
+    "Results": [
+      {
+        "Text": "el lunes 21, entre las 9:30 y las 3:00 p.m.",
+        "Start": 25,
+        "End": 67,
+        "TypeName": "datetimeV2.datetimerange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(XXXX-WXX-1T09:30,XXXX-WXX-1T15:00,PT5H30M)",
+              "type": "datetimerange",
+              "start": "2019-01-21 09:30:00",
+              "end": "2019-01-21 15:00:00"
+            },
+            {
+              "timex": "(XXX
```

Comment:
This is incorrect. The last trimester/quarter are the last 3 months of the year.
This resolution is the 3rd trimester.